src/Question.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { Link, BrowserRouter as Router } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+class Question extends Component {
```

Comment:
As far as I know stateless functional component is not deprecated. And I don't consider using (putting) PureComponent every where without of a real need.
I don't see a hard necessity in a PureComponent here.